docker-images/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG PYTHON_VERSION=3.11.7
+FROM python:${PYTHON_VERSION}-slim as base
```

Comment:
you can remove `ARG` if you're not going to build this docker image with different versions of python